bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/util/ByteStringer.java

```diff
@@ -49,13 +49,19 @@ private ByteStringer() {
 
   /**
    * Wraps a byte array in a {@link ByteString} without copying it.
+   * @param array an array object.
+   * @return this
```

Comment:
`@return this isn't right here`

Reply:
Updated the line with:
`@return {@link ByteString} based on runtime copy flag.`